cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md

```diff
@@ -107,6 +107,38 @@ csrfToken = hmac + "." + randomValue // Add the `randomValue` to the HMAC hash t
 response.setCookie("csrf_token=" + csrfToken + "; Secure") // Set Cookie without HttpOnly flag
 ```
 
+Below is an example in psuedo-code that demonstrates validation of the CSRF token once it is sent back from the client:
```

Comment:
"pseudo-code"

Reply:
What do you mean by putting scare quotes around the word "pseudo-code"? Do you mean that you do not think the code presented is pseudo-code? Or is it the opposite and you are highlighting that the code is pseudo-code and not real code?

Reply:
Sorry I meant there was a typographic error: "psuedo-code" → "pseudo-code"

Reply:
Ah! Thanks. 😊